spec/errors.md

```diff
@@ -34,7 +34,7 @@ SHOULD prioritise _Syntax Errors_ and _Data Model Errors_ over others.
 
 When an error occurs within a _selector_,
 the _selector_ MUST NOT match any _variant_ _key_ other than the catch-all `*`
-and a _Resolution Error_ or a _Selection Error_ MUST be emitted.
+and a _Resolution Error_ or a _Message Function Error_ MUST be emitted.
```

Comment:
Selection Error is a sub-species of Message Function Error, right? Any reason to expand?

Reply:
Leaving it as Message Function allows for e.g. a lazy implementation to only discover at this point that one of its options that are only relevant for selection is badly formed, and so emit a more useful Bad Option error instead.